packages/@vue/cli-plugin-eslint/package.json

```diff
@@ -32,6 +32,6 @@
   },
   "peerDependencies": {
     "@vue/cli-service": "^3.0.0 || ^4.0.0 || ^5.0.0-0",
-    "eslint": ">=7.5.0"
+    "eslint": "^8.0.0"
```

Comment:
I can revert no problem. I was wondering: how to test that there is no issue with eslint v7? Duplicate all tests for v7 and v8? Or do we just assume it works?

Reply:
For now, it should work, because we didn't touch any code in `index.js` or `lint.js`.

In the future, yes, a dedicated test setup for ESlint 7 is needed.

Reply:
All right let's keep `>=7.5.0` for now then.
PR updated.